.happy/config.json

```diff
@@ -3,7 +3,7 @@
     "terraform_version": "0.13.5",
     "default_env": "rdev",
     "app": "genepi",
-    "default_compose_env": ".env.ecr",
+    "default_compose_env_file": ".env.ecr",
```

Comment:
Why change this parameter name?

Reply:
default_env (environment name) and default_compose_env (an environment file name) have completely different semantics, we've added some clarity in the golang version.